static/app/views/issueList/groupListBody.tsx

```diff
@@ -99,7 +99,7 @@ function GroupList({
   const topIssue = groupIds[0];
   const canSelect = !useMedia(
     `(max-width: ${
-      isSavedSearchesOpen ? theme.breakpoints.large : theme.breakpoints.small
+      isSavedSearchesOpen ? theme.breakpoints.xlarge : theme.breakpoints.medium
```

Comment:
This matches https://github.com/getsentry/sentry/blob/78ea0119180fb9321beff3bfafb19ae08afc0fd3/static/app/views/issueList/actions/index.tsx#L179